docs/usage/folders.md

```diff
@@ -43,7 +43,8 @@ ones not returned by default.
 <!-- sample get_folders_id -->
 ```python
 folder = client.folder(folder_id='22222').get()
-print('Folder "{0}" has {1} items in it'.format(folder.name, folder.item_collection.total_count))
+print('Folder "{0}" has {1} items in it'
```

Comment:
our general Python coding style would have this line as 

    print('Folder "{0}" has {1} items in it'.format(
        folder.name,
        folder.item_collection.total_count,
    ))

Reply:
That is actually even better. I'll send an update.